features/api/adherents.feature

```diff
@@ -138,3 +138,91 @@ Feature:
       "lastName":"Picard"
     }
     """
+
+  Scenario: As a non logged-in user I can not set a nickname
+    Given I send a "PUT" request to "/api/adherents/me/anonymize"
+    Then the response status code should be 401
+
+  Scenario: As a logged-in user I can not set a nickname of another person
+    Given I am logged as "jacques.picard@en-marche.fr"
+    When I send a "PUT" request to "/api/adherents/me/anonymize" with body:
```

Comment:
Can you please add a header `Accept: application/json` as soon as you expect a json response ? (and not redirection etc)

just to avoid more conflicts in my PR regarding jwt authentification

Reply:
Done.